diff -rau rustc-1.62.0-src-orig/compiler/rustc_driver/src/lib.rs rustc-1.62.0-src/compiler/rustc_driver/src/lib.rs
--- rustc-1.62.0-src-orig/compiler/rustc_driver/src/lib.rs	2022-06-27 15:37:07.000000000 +0200
+++ rustc-1.62.0-src/compiler/rustc_driver/src/lib.rs	2022-07-12 06:39:44.073910401 +0200
@@ -11,6 +11,9 @@
 #![recursion_limit = "256"]
 #![allow(rustc::potential_query_instability)]
 
+#[link(name="z")]
+extern {}
+
 #[macro_use]
 extern crate tracing;
 
